lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -46,6 +49,8 @@ class ProgressTabView(RetrieveAPIView):
             is_requestable: (bool) true if status is requesting and request_cert_url is not None
             msg: (str) message for the certificate status
             title: (str) title of the certificate status
+        credit_course_requirements:
+        credit_purchase_url: (str) the url to the support docs for purchasing a credit
```

Comment:
If this is a URL to support documentation, `credit_purchase_url` feels misleading. I thought this would be redirecting to where someone would go to be purchasing their credit.